src/index.css

```diff
@@ -0,0 +1,398 @@
+@media (max-width: 991.98px) {
+  .offcanvas-collapse {
+    position: fixed;
+    top: 56px; /* Height of navbar */
+    bottom: 0;
+    left: 100%;
+    width: 100%;
+    padding-right: 1rem;
+    padding-left: 1rem;
+    overflow-y: auto;
+    visibility: hidden;
+    background-color: #f8f9fa;
+    transition: transform 0.3s ease-in-out, visibility 0.3s ease-in-out;
+  }
+  .offcanvas-collapse.open {
+    visibility: visible;
+    transform: translateX(-100%);
+  }
+}
+
+.bg-custom-light {
+  background-color: #e7e7e7;
+}
+
+.bg-dim {
+  background-color: rgba(4, 4, 4, 0.261);
+}
+
+.cover {
+  object-fit: cover;
+}
+
+.opacity-6 {
+  opacity: 0.6;
+}
+
+.opacity-5 {
+  opacity: 0.5;
+}
+
+.image-vertical-scroller {
+  overflow-y: auto;
+  max-height: 460px;
+  -webkit-overflow-scrolling: touch;
+  -ms-overflow-style: none; /* IE and Edge */
+  scrollbar-width: none;
+}
+
+.image-vertical-scroller::-webkit-scrollbar {
+  display: none;
+}
+
+.h-scroller {
+  position: relative;
+  z-index: 2;
+  height: 3.75rem;
+  overflow-y: hidden;
+}
+
+.h-scroller .nav {
+  display: flex;
+  flex-wrap: nowrap;
+  padding-bottom: 1rem;
+  margin-top: -1px;
+  overflow-x: auto;
+  color: rgba(255, 255, 255, .75);
+  text-align: center;
+  white-space: nowrap;
+  -webkit-overflow-scrolling: touch;
```

Comment:
@rahulmeghwal do you have any tactics around avoiding having to define all the engine-specific CSS properties vs just the standard CSS spec?

Reply:
Have put in some tactics around the height of the text displayed and other properties to avoid engine-specific css. (Removed some unnecessary css as well)